python/lsst/ci/hsc/validate.py

```diff
@@ -189,6 +189,14 @@ class SfmValidation(Validation):
     def validateSources(self, dataId):
```

Comment:
Oh wait, you can ignore my previous comment.  I see that there are more tests in this function.